operator/controllers/kyma_controller.go

```diff
@@ -183,7 +180,7 @@ func (r *KymaReconciler) HandleProcessingState(ctx context.Context, kyma *v1alph
 	var err error
 	var modules parsed.Modules
 	// these are the actual modules
-	modules, err = r.GetModules(ctx, kyma, false)
```

Comment:
This false check means it will not actually try to check for outdated templates during processing which I think resolves the issue discussed above